src/Utility/Text.php

```diff
@@ -649,6 +649,20 @@ public static function truncate($text, $length = 100, array $options = [])
     }
 
     /**
+     * Truncate text with specified width.
+     *
+     * @param string $text String to truncate.
+     * @param int $length Length of returned string, including ellipsis.
+     * @param array $options An array of HTML attributes and options.
+     * @return string Trimmed string.
+     * @see \Cake\Utility\Text::truncate()
+     */
+    public static function truncateWidth($text, $length = 100, array $options = [])
```

Comment:
What about `widthTruncate()`? If it also sounds strange, I am willing to rename the method to `truncateWithWidth`.